runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttBindingConfig.java

```diff
@@ -223,7 +223,7 @@ private Function<String, String> asAccessor(
                 Pattern.compile(config.pattern.replace("{credentials}", "(?<credentials>[^\\s]+)"))
                     .matcher("");
 
-            injector = orElseIfNull(injector, connect ->
+            accessor = orElseIfNull(accessor, connect ->
```

Comment:
Do we need `orElseIfNull` for the `accessor` either?
Suspect not, so let's update and remove `orElseIfNull` as well, to eliminate dead code.